package.json

```diff
@@ -30,6 +30,7 @@
     "eslint-plugin-no-relative-import-paths": "^1.5.3",
     "jsdom": "^22.1.0",
     "lerna": "^6.6.1",
+    "memfs": "^4.17.2",
```

Comment:
the root package.json should only be the dev tools that is used across the repo or the mono repo scripts.

`memfs` is a dev dependency used by the generator package, let's move it to the that package instead of monorepo root.

```sh
npx lerna add memfs --scope=@salesforce/pwa-kit-create-app --dev
```